Applications/AdminCli/src/AdminCli/Commands/Clients/CreateClientCommand.cs

```diff
@@ -53,9 +53,10 @@ private async Task CreateClient(string dbProvider, string dbConnectionString, st
     {
         var mediator = _serviceLocator.GetService<IMediator>(dbProvider, dbConnectionString);
 
-        var response = await mediator.Send(new Modules.Devices.Application.Clients.Commands.CreateClient.CreateClientCommand(clientId, displayName, clientSecret, defaultTier, maxIdentities), CancellationToken.None);
+        var response = await mediator.Send(new Modules.Devices.Application.Clients.Commands.CreateClient.CreateClientCommand(clientId, displayName, clientSecret, defaultTier, maxIdentities),
+            CancellationToken.None);
 
         Console.WriteLine(JsonSerializer.Serialize(response, JSON_SERIALIZER_OPTIONS));
-        Console.WriteLine("Please note the secret since you cannot obtain it later.");
+        Console.WriteLine(@"Please note the secret since you cannot obtain it later.");
```

Comment:
Why?

Reply:
To be honest: I have no idea. 😆 Rider complained about it ("Localizable interpolated string") and I didn't want to start a discussion about it. 😜